tests/std/tests/P0024R2_parallel_algorithms_for_each/test.cpp

```diff
@@ -47,27 +47,35 @@ const auto call_only_once = [](atomic<bool>& b) { assert(!b.exchange(true)); };
 
 const auto atomic_identity = [](atomic<bool>& b) { return b.load(); };
 
-template <template <class...> class Container>
-void test_case_for_each_parallel(const size_t testSize) {
+template <typename ExecutionPolicy, template <class...> class Container>

```

Comment:
Oh, I see - you're passing a function pointer below, instead of directly calling it. This is fine, although it *could* be restructured as a `struct` templated on an explicitly provided `Container`, with a function call operator deducing `ExecutionPolicy`. No change requested.